iree/compiler/Utils/TracingUtils.h

```diff
@@ -34,8 +34,10 @@ struct PassTracing : public PassInstrumentation {
   // Note: we could also trace pipelines and analyses.
 
   void runBeforePass(Pass *pass, Operation *op) override {
-    std::string passName = pass->getName().str();
-    IREE_TRACE_ZONE_BEGIN_NAMED_DYNAMIC(z0, passName.data(), passName.size());
+    static std::string fileName = __FILE__;
```

Comment:
is this needed, or can you just do __FILE__ and strlen(__FILE__)? (the strlen should be compile-time evaluated)

Reply:
Ah, I thought it might work, but didn't try it. Tried it now and it works :)